.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 # JetBrains IDEs
 /.idea
+
+# Pyright configuration
+pyrightconfig.json
```

Comment:
Why would we ignore a config file?
No need to ignore something we don't even use, but if we did, we'd want to commit it.
```suggestion
```